### ISSUE TITLE
Example doesn't work in Contao 4.9.20

### ISSUE THREAD

**Issue by stefanschleich** [terminal42/contao-url-rewrite#29]
I just tried the following example from the docs in a Contao 4.9.20 with the extension version 1.5.0:

```
Typ: einfach
Pfad-Einschränkung: find/{address}
Antwort-Statuscode: 303 See Other
Antwort-Umleitungs-URL: https://www.google.com/maps?q={address}
---
Resultat: domain.tld/find/Schweiz → https://www.google.com/maps?q=Schweiz
```

The result is:

`domain.tld/find/test → https://www.google.com/maps?q={address&#125;`

Same for the news example: 

`domain.tld/news/test → domain.tld/news/{news&#125;`

**Comment by fritzmg** [comment]
Duplicate of https://github.com/terminal42/contao-url-rewrite/issues/28 I think.

**Comment by stefanschleich** [comment]
I understood #28 to be a problem with custom made inserttags, but I just used the examples provided with the extension. 

Is this a bug or can i solve it on my own somehow?

**Comment by fritzmg** [comment]
It's a bug.

**Comment by stefanschleich** [comment]
Thanks. I worked around it by configuring the redirect via the config.yml.

**Comment by fritzmg** [comment]
This should be fixed in Contao 4.9.22. @stefanschleich can you confirm?

**Comment by rabauss** [comment]
Should be fixed with https://github.com/contao/contao/pull/3407
